lib/frederick_api/v2/helpers/backgroundable_parser.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module FrederickAPI
+  module V2
+    module Helpers
+      # Custom Parser for parsing BackgroundJob resources for FrederickAPI V2
+      class BackgroundableParser < ::JsonApiClient::Parsers::Parser
+        def self.parse(klass, response)
+          result_set = super(klass, response)
+          return result_set unless result_set&.first&.type == 'background_jobs'
+          result_set = super(::FrederickAPI::V2::BackgroundJob, response)
+          result_set&.first&.meta = { headers: response.headers, status: response.status }
```

Comment:
I think it is possible for meta to serialized, so I would be wary of overriding it like this. If what you want to keep track of is the response itself, why not just ass a attr_accessor for the response on the BackgroundJobResource?